sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/README.md

```diff
@@ -0,0 +1,130 @@
+# Azure Partition Manager client library for Java using Storage Blobs.
+
+[Source code][source_code] | [API reference documentation][api_documentation] | [Product
+documentation][event_hubs_product_docs] | [Samples][sample_examples]
+
+## Getting started
+
+### Prerequisites
+
+- Java Development Kit (JDK) with version 8 or above
+- [Maven][maven]
+- Microsoft Azure subscription
+    - You can create a free account at: https://azure.microsoft.com
+- Azure Event Hubs instance
+    - Step-by-step guide for [creating an Event Hub using the Azure Portal][event_hubs_create]
+- Azure Storage account
+    - Step-by-step guide for [creating a Storage account using the Azure Portal][storage_account]
+### Adding the package to your product
+
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-messaging-eventhubs-checkpointstore-blob</artifactId>
+    <version>1.0.0-preview.1</version>
+</dependency>
+```
+
+## Key concepts
+
+## Examples
+- [Consume events from all Event Hub partitions][sample_event_processor]
+
+### Consume events using an Event Processor
+
+To consume events for all partitions of an Event Hub, you'll create an [`EventProcessor`][source_eventprocessor] for a
+specific consumer group. When an Event Hub is created, it provides a default consumer group that can be used to get
+started.
+
+The [`EventProcessor`][source_eventprocessor] will delegate processing of events to a
+[`PartitionProcessor`][source_partition_processor] implementation that you provide, allowing your application to focus on the
+business logic needed to provide value while the processor holds responsibility for managing the underlying consumer operations.
+
+In our example, we will focus on building the [`EventProcessor`][source_eventprocessor], use the 
+[`BlobPartitionManager`][source_inmemorypartitionmanager], and a `PartitionProcessor` implementation that logs
+events received and errors to console.
+
+```java
+class Program {
+    public static void main(String[] args) {
+        EventProcessor eventProcessor = new EventHubClientBuilder()
+            .connectionString("<< CONNECTION STRING FOR THE EVENT HUB INSTANCE >>")
+            .consumerGroupName("<< CONSUMER GROUP NAME>>")
+            .partitionProcessorFactory(SimplePartitionProcessor::new)
+            .partitionManager(new BlobPartitionManager(containerAsyncClient))
```

Comment:
Is it weird to call it a checkpoint store (i.e. `azure-messaging-eventhubs-checkpointstore-blob`) but to refer to it in code as a partition manager? It kind of feels this project makes more sense as a `azure-messaging-eventhubs-partitionmanager-blob`.

Reply:
We will update the name of PartitionManager in upcoming preview. We wanted to get the name of the package finalized and PartitionManager will be renamed in the next release.